challenge_7/cpp/rbrt/src/[cpp]Challenge_7.cpp

```diff
@@ -0,0 +1,50 @@
+#include <iostream> 
+
+ 
+using namespace std;
+
+
+ int main()
+{
+ 		int n; 
+		cout<< "Please insert the array length N >> ";
+		cin>>n;
+
+		int x[1000]={0};
+
+		cout<< "Please insert an array from 0 to N>> ";
```

Comment:
Your job is to generate this list or use the test cases we have. What if the list is of size 1,000,000?